test/test_all_basic.rb

```diff
@@ -28,6 +28,27 @@
   require file
 end
 
+module Minitest
+  module Assertions
+    def assert_nothing_raised
+      yield
+    end
+
+    def assert_block
+      assert(yield)
+    end
+
+    alias assert_nothing_thrown assert_nothing_raised
+    alias assert_raise assert_raises
+    alias assert_raise_message assert_raises
```

Comment:
Do we still need this alias?

Reply:
No we don't! Good catch. Updated.

Reply:
Woops, pushed it to a different branch. Now it's actually updated.